Tabloid/Controllers/TagController.cs

```diff
@@ -0,0 +1,85 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using System.Collections.Generic;
+using Tabloid.Repositories;
+using Microsoft.VisualBasic;
+using System.Security.Claims;
+using Tabloid.Models.ViewModels;
+using Microsoft.AspNetCore.Authorization;
+using System;
+using Tabloid.Models;
+
+namespace Tabloid.Controllers
+{
+    [Route("api/[controller]")]
```

Comment:
This controller should be protected with teh `[Authorize]` attribute to require logged in user requests.